test/onigumo_test.exs

```diff
@@ -3,19 +3,18 @@ defmodule OnigumoTest do
   import Mox
 
   @urls [
-    "http://onigumo.local/hello.html",
-    "http://onigumo.local/bye.html"
+    {"http://onigumo.local/hello.html", "aHR0cDovL29uaWd1bW8ubG9jYWwvaGVsbG8uaHRtbA"},
+    {"http://onigumo.local/bye.html", "aHR0cDovL29uaWd1bW8ubG9jYWwvYnllLmh0bWw"}
   ]
-  @output_path "body.html"
 
   setup(:verify_on_exit!)
 
   @tag :tmp_dir
   test("download a single URL", %{tmp_dir: tmp_dir}) do
     expect(HTTPoisonMock, :get!, &get!/1)
 
-    input_url = Enum.at(@urls, 0)
-    output_path = Path.join(tmp_dir, @output_path)
+    input_url = urls() |> Enum.at(0)
+    output_path = Path.join(tmp_dir, "body.html")
```

Comment:
Now, the file name is arbitrary and is used only on this very place. Thus switching to a literal.